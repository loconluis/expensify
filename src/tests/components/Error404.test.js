import React from 'react';
import { shallow } from 'enzyme';
import Error404 from '../../components/Error404';

test('Should render ERROR404 page correctly', () => {
  const wrapper = shallow(<Error404 />);
  expect(wrapper).toMatchSnapshot();
})