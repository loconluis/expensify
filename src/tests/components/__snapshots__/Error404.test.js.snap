// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Should render ERROR404 page correctly 1`] = `
<div>
  Error 404. Not found! - 
  <Link
    replace={false}
    to="/"
  >
    Go Home
  </Link>
</div>
`;
